src/string_util.c

```diff
@@ -2,6 +2,7 @@
 #include "string_util.h"
 #include "text.h"
 #include "strings.h"
+#include "malloc.h"
```

Comment:
Is this used?

Reply:
Ah, yes it is unused.
I had the function in this file at the start and forgot to remove this.